src/models/server.ts

```diff
@@ -1,12 +1,12 @@
 import type { BaseModel } from './base';
 import type { ChannelsInterface } from './channels';
 import type { MessagesInterface } from './messages';
-import type { BindingsMixinInterface, DescriptionMixinInterface, ExtensionsMixinInterface } from './mixins';
+import type { BindingsMixinInterface, DescriptionMixinInterface, ExtensionsMixinInterface, TagsMixinInterface } from './mixins';
 import type { OperationsInterface } from './operations';
 import type { ServerVariablesInterface } from './server-variables';
 import type { SecurityRequirementsInterface } from './security-requirements';
 
-export interface ServerInterface extends BaseModel, DescriptionMixinInterface, BindingsMixinInterface, ExtensionsMixinInterface {
+export interface ServerInterface extends BaseModel, DescriptionMixinInterface, BindingsMixinInterface, ExtensionsMixinInterface, TagsMixinInterface {
```

Comment:
Would u mind adding this change into `parser-api` PR https://github.com/asyncapi/parser-api/pull/71?